.eslintrc.js

```diff
@@ -14,7 +14,37 @@ module.exports = {
     '@elastic/eslint-config-kibana',
     'plugin:@elastic/eui/recommended',
     'plugin:react-hooks/recommended',
+    "eslint:recommended",
+    "plugin:cypress/recommended",
+    "plugin:import/recommended",
+    "prettier"
   ],
+  env: {
+    'cypress/globals': true,
+  },
+  plugins: [
+    'cypress',
+  ],
+  rules: {
+    '@osd/eslint/no-restricted-paths': [
```

Comment:
what does this do?

Reply:
this restricts certain import paths based on specified patterns: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d8cbc170d1b8f07b0c2938ce32fe3ff125911030/packages/osd-eslint-plugin-eslint/rules/no_restricted_paths.js

I borrowed this block from [core](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.eslintrc.js#L269) and [security plugin](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/.eslintrc.js#L29) just thinking we could directly benefit from it if we have existing issues or having issues in the future